server/src/main/java/org/opensearch/action/search/TransportSearchAction.java

```diff
@@ -475,6 +475,7 @@ private void executeRequest(
         } else {
             searchRequestOperationsListener = null;
         }
+        searchRequestOperationsListener.onRequestStart(relativeStartNanos);
```

Comment:
Passing a startTime doesn't make sense. If we are calling onRequestStart(), it ideally means we are calling it when the request starts, so startTime can be tracked by the listener itself (which is slow log here).

Reply:
Fixed it